src/analyze/analyze.c

```diff
@@ -2346,29 +2347,15 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
 
                 case ARG_MAN:
-                        if (optarg) {
-                                r = parse_boolean(optarg);
-                                if (r < 0)
-                                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
-                                                               "Failed to parse --man= argument.");
-
-                                arg_man = r;
-                        } else
-                                arg_man = true;
-
+                        r = parse_boolean_argument("--man", optarg, &arg_man);
```

Comment:
hmm, first argument should probably be suffixed with "=", it's a switch that expects an argument after all, and the old error message *did* suffix it like that.

i.e. we usually have this rule to suffix cmdline switches and config settings that take an argument with "=", to indicate that.

Reply:
You're right. I added this now in the places where the arg is required (but not where not).